Scripts/script-listExecutedCommands.yml

```diff
@@ -0,0 +1,64 @@
+commonfields:
+  id: listExecutedCommands
+  version: -1
+name: listExecutedCommands
+script: |-
+  var fixNewLines = function(str) {
+      var tmp = str.split('\n').join('<br>');
```

Comment:
You can just do 
`return str.split('\n').join('<br>').split('\r').join(' ');`

Also, why not using replace?